hil/api.py

```diff
@@ -1227,6 +1227,16 @@ def list_headnode_images():
     return json.dumps(valid_imgs)
 
 
+# Extension code #
+#################
+@rest_call('GET', '/active_extensions', Schema({}))
```

Comment:
I missed that you were doing that in CLI. In that case you can ignore this comment.